kedro_azureml/generator.py

```diff
@@ -280,8 +302,16 @@ def _connect_commands(self, pipeline: Pipeline, commands: Dict[str, Command]):
                     parent_outputs = invoked_components[output_from_deps.name].outputs
                     azure_output = parent_outputs[sanitized_input_name]
                     azure_inputs[sanitized_input_name] = azure_output
+                # 2. try to find AzureMLFolderDataSet in catalog
+                elif node_input in self.catalog.list() and isinstance(
+                    ds := self.catalog._get_dataset(node_input), AzureMLFolderDataSet
+                ):
+                    azure_inputs[sanitized_input_name] = Input(
+                        # TODO: add versioning
+                        path=f"{ds._azureml_dataset}@latest"
```

Comment:
the `@latest` could also be taken from the `ds` as you're already accessing it here.

Reply:
This is a placeholder until the versioning is implemented. Do you have any opinion on how this should be done (see PR description)?

Reply:
I have updated the PR to use the `--load-versions` CLI argument in the same way Kedro does